apps/i18n/common/en_us.json

```diff
@@ -195,6 +195,7 @@
   "shareWarningsAge": "Please provide your age below and click OK to continue.",
   "shareWarningsMoreInfo": "More Info",
   "shareWarningsStoreData": "This app built on Code Studio stores data that could be viewed by anyone with this sharing link, so be careful if you are asked to provide personal information.",
+  "showBlock": "Get a block",
```

Comment:
That said, some experiments may have different results depending on the locale. E.g., I could see this experiment having results that vary by locale (as a proxy for culture).